test/Autofac.Integration.Wcf.Test/AutofacInstanceContextFixture.cs

```diff
@@ -63,6 +64,43 @@ public void Resolve_RequiresServiceImplementationData()
             Assert.Throws<ArgumentNullException>(() => context.Resolve(null));
         }
 
+        [Fact]
```

Comment:
The code in the class handles `.Modules` being null; it might be good to have a test showing that's handled.